src/pages/ServerRules.js

```diff
@@ -23,46 +22,51 @@ class ServerRules extends Component {
 						<h2>General Composure</h2>
 						<ol>
 							<li>
-								Maintain a casual and positive atmosphere at all times. Be
-								respectful of all players. Harassment and bullying are strictly
-								not allowed.
+								Ensure that you are respectful to all players and are
+								maintaining a calm and positive atmosphere while on our server.
+								Any forms of harassment or bullying are strictly forbidden.
+								Don't be toxic.
```

Comment:
If we're going to include the text "Don't be toxic." we should be more specific on what we consider to be toxicity.

Reply:
edited